ipu-plugin/pkg/ipuplugin/ipuplugin.go

```diff
@@ -86,9 +135,13 @@ func (s *server) Run() error {
 	}
 
 	if s.mode == types.IpuMode {
+		// Wait for the infrap4d connection to come up
+		if _, err := waitForInfraP4d(); err != nil {
```

Comment:
Also, how are you testing your changes.

Reply:
By starting the VSP pod and then starting the p4 container pod after a little while. The VSP pod gets stuck on this loop waiting for the p4 container to show up and then continues once it finds a forwarding config

Reply:
After it comes up, I verify that the init time p4 rules get written and the OVS bridge comes up with the interfaces